engine/src/main/java/org/terasology/world/viewer/picker/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2013 MovingBlocks
```

Comment:
Hardly a problem but it looks like we've got some outdated headers you're cloning